BunqSdk.Tests/BunqSdkTestBase.cs

```diff
@@ -13,18 +12,18 @@ public class BunqSdkTestBase
         /// <summary>
         /// Name of the context configuration file.
         /// </summary>
-        private const string FILENAME_CONTEXT_CONF = "../../../bunq-test.conf";
+        private const string FilenameContextConf = "../../../bunq-test.conf";
 
         /// <summary>
         /// Device description used for tests.
         /// </summary>
-        private const string DEVICE_DESCRIPTION_TEST = "Csharp unit test";
+        private const string DeviceDescriptionTest = "Csharp unit test";
 
         /// <summary>
         /// Configuration items.
         /// </summary>
-        private static readonly string API_KEY = Config.GetApiKey();
-        private static readonly string[] FIELD_PERMITTED_IPS = Config.GetPermittedIps();
+        private static readonly string ApiKey = Config.GetApiKey();
+        private static readonly string[] FieldPermittedIps = Config.GetPermittedIps();
```

Comment:
Plural -> FieldAllPermittedIp

Reply:
@OGKevin @sandervdo should the Config method name be renamed too?

Reply:
@antao yes please 👏

Reply:
@antao yes please. 😄  When you find more occurrences of plurals feel free to change them! 👍  Thanks a lot!